code/Pancake Sorting/pancake_sort.c

```diff
@@ -0,0 +1,108 @@
+/*
+    Pancake Sorting Algorithm
+*/
+
+#include<stdio.h>
+
+// Print the sorted array
+
+void print_ans(int a[],int n)
+{
+    printf("Sorted Array\n");
+    
+    for(int i=0;i<n;i++)
```

Comment:
Reformat this code with clang-format 4 spaces for both C++ and C.

Reply:
Do i have to add 4 spaces before every line in clang-format 4 spaces

Reply:
??